7.1.11-fpm-alpine3.4/Dockerfile

```diff
@@ -48,6 +48,11 @@ ENV MAILHOG_VERSION v0.1.9
 RUN curl -fSL "https://github.com/mailhog/MailHog/releases/download/${MAILHOG_VERSION}/MailHog_linux_amd64" -o /usr/local/bin/mailhog \
     && chmod +x /usr/local/bin/mailhog
 
+# Install ldap module.
+RUN apk add --no-cache ldb-dev libldap openldap-dev
+RUN docker-php-ext-configure ldap
```

Comment:
The `docker-configure-extension ldap` command creates the file `/usr/local/etc/php/conf.d/docker-php-ext-ldap.ini` which enables the extension by default. If you want to keep this extension optional you have to remove this command.

Reply:
so we just need the `docker-php-ext-install` right ?